libc/unistd_uids.c

```diff
@@ -110,7 +112,7 @@ TEST(unistd_uids, setpuids_setsid)
 	TEST_ASSERT_GREATER_THAN_INT(0, parent[1]);  /* nonzero pgid */
 	TEST_ASSERT_GREATER_THAN_INT(0, parent[2]);  /* nonzero sid */
 	TEST_ASSERT_EQUAL_INT(parent[0], parent[1]); /* pid == pgid */
-	TEST_ASSERT_EQUAL_INT(parent[1], parent[2]); /* pgid == sid */
+	/* we don't know the initial session leader, so we don't assert the exact value */
```

Comment:
This comment is on place regarding this commit, but is out of context in this file. Better use something like:
```suggestion
	/* we don't check parent pgid against sid as we don't know who was parents session leader */
```